enhancements/windows-containers/disconnected-support.md

```diff
@@ -0,0 +1,439 @@
+- [x] Test plan is defined
+- [x] Operational readiness criteria is defined
+- [x] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+The goal of this enhancement is to support Windows Containers in environments with restricted networks where
+hosts are intentionally impeded from reaching the internet, also known as disconnected or "air-gapped" clusters.
+Currently, Windows nodes configured by the [Windows Machine Config Operator](https://github.com/openshift/windows-machine-config-operator) 
+(WMCO) rely on `containerd``, the OpenShift-managed container runtime, to pull workload images. Also, the WMCO today
+is required to make an external request outside the cluster's internal network to pull the [pause image](https://kubernetes.io/docs/concepts/windows/intro/#pause-container).
+To support disconnected environments, all images need to be pulled from air-gapped mirror registries, whether that be 
+the OpenShift internal image registry or other private registries.
+
+There already exists a protocol for users to publish [registry mirroring configuration](https://docs.openshift.com/container-platform/4.14/openshift_images/image-configuration.html#images-configuration-registry-mirror_image-configuration),
+namely `ImageDigestMirrorSet` (IDMS), `ImageTagMirrorSet` (ITMS) cluster resources. 
+These are consumed by OpenShift components like the Machine Config Operator (MCO) to apply the settings to Linux
+control-plane and worker nodes, Windows worker nodes do not currently consume or respect mirror registry settings when
+pulling images. This effort will work to plug feature disparity by making the WMCO aware of mirror registry settings at
+operator install time and reactive during its runtime.
+
+
+## Motivation
+
+The motivation here is to expand the Windows containers production use case, enabling users to add Windows nodes and run
+workloads easily and successfully in disconnected clusters. This is an extremely important ask for customer environments
+where Windows nodes must pull images from air-gapped registries for security reasons.
+The added benefit here is that the changes proposed in this enhancement are not restricted to disconnected environments;
+users in __connected__ clusters can take advantage of this mirror registry support. This would allow better performance
+as images can be stored in registries closer to the production environment, reducing pull times for notoriously large 
+Windows container images, in addition the security benefits of restricted networks.
+
+### Goals
+
+* Create a mechanism for WMCO to consume global mirror registry settings from existing platform resources (IDMS/ITMSes),
+  such that it enables Windows nodes to pull images in disconnected environments
+* Configure the mirror registry settings in the containerd runtime, a WMCO-managed component on Windows nodes 
```

Comment:
Is this mirror registry settings or just registry settings from a containerd perspective?

Reply:
from containerd's perspective these are just registry settings, but WMCO will only be setting those registry settings related to mirrors